termwiz/src/escape/parser/mod.rs

```diff
@@ -81,11 +84,39 @@ impl Parser {
     }
 
     pub fn parse<F: FnMut(Action)>(&mut self, bytes: &[u8], mut callback: F) {
-        let mut perform = Performer {
-            callback: &mut callback,
-            state: &mut self.state.borrow_mut(),
-        };
-        self.state_machine.parse(bytes, &mut perform);
+        let tmux_state: bool = self.state.borrow().tmux_state.is_some();
```

Comment:
nit: rename for clarity:

```suggestion
        let is_tmux_mode_active = self.state.borrow().tmux_state.is_some();
```